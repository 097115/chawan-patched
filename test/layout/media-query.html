<style>
@mediA (WiDtH >= 1px) and (HeiGhT < 10000px) {
	#red { color: red }
}
@media screen and (WiDtH >= 1px) {
	#green { color: green }
}
@media (width >= 1px 2px) {
	#red { color: blue }
}
</style>
<div id=red>red</div>
<div id=green>green</div>
