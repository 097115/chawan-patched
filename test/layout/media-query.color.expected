[38;2;255;40;40mred[39m
[38;2;41;169;42mgreen[39m
